SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/accounts/view/AccountsViewModel.kt

```diff
@@ -77,17 +78,19 @@ class AccountsViewModel : ViewModel() {
 
                 val balanceValue = BigDecimal(balance.platformBalance ?: JavaBigDecimal(0))
                 val balanceValueFormatted = AssetPipe.transform(balanceValue, assetDecimals, "trade")
+                val balanceValueFormattedString = balanceValueFormatted.toPlainString()
 
                 val buyPrice = BigDecimal(price?.buyPrice ?: JavaBigDecimal(0))
                 val buyPriceFormatted = BigDecimalPipe.transform(buyPrice, pairAsset!!)
 
-                val accountBalanceInFiat = balanceValueFormatted.times(buyPrice)
+                val accountBalanceInFiat = balanceValueFormatted.times(buyPrice).setScale(2)
```

Comment:
Is `.setScale` the way we want to handle formatting? Seems we should use the currency pipe here, no?

Reply:
Im using here cause the Asset formatting gives you the exact value with n number of decimals, but the design requirement needs in some labels only 2 decimals, for that reason this value scale is only modifed in the ViewModel